README.md

```diff
@@ -128,6 +128,7 @@ When starting `x16emu` without arguments, it will pick up the system ROM (`rom.b
 * `-sound <device>` can be used to specify the output sound device. If 'none', no audio is generated.
 * `-abufs` can be used to specify the number of audio buffers (defaults to 8 when using the SD card, 32 when using HostFS). If you're experiencing stuttering in the audio, try increasing this number. This will result in additional audio latency though.
 * `-via2` installs the second VIA chip expansion at $9F10.
+* `-user-peripheral <my_peripheral.{so|dll}>` connects an emulated user peripheral built as a dynamically linked library. implies `-via2`
```

Comment:
macOS uses `.dylib`. The extension also isn't included in `main.c` (and I'm not sure whether it should be).